src/routes/controllers/articles.controller.js

```diff
@@ -36,7 +37,7 @@ export async function createArticle(req, res) {
       return errorResponse(res, 401, 'jwt must be provided');
```

Comment:
I believe there is a middleware called ```checkAuthorizedUser()``` that you can apply to the your route so you can keep the codes `DRY`

Reply:
OK, I will check that out. thanks.